spring-cloud-aws-autoconfigure/src/main/java/org/springframework/cloud/aws/autoconfigure/context/ContextStackAutoConfiguration.java

```diff
@@ -33,6 +33,7 @@
  * @author Agim Emruli
  */
 @Configuration
+@ConditionalOnAwsCloudEnvironment
```

Comment:
H-m...
I would like to see that discussion on Gitter, because allmost all these changes are breaking for me.
Looks like I fully can't get a gain from AWS auto-configuration in my Boot apps if I don't use EC2. :cry: